### PR TITLE
fix(DB/gameobject): Add treasures to Bloodmyst Isle

### PR DESCRIPTION
**Pull request by Zoidwaffle**
## Changes Proposed:
-  Increase amount of treasures from 0 -> 28 (chests/giant clams)
- Add them all to a single pool, size 10
- Nodes respawn, randomly, after 5 minutes if taken 

## Issues Addressed:
- Based upon data from Gathermate - I tried not to spawn similar objects too close, not spawn things in wrong locations etc..

## Tests Performed:
- Applied SQL
- All locations visited...

## How to Test the Changes:

- Learn the proper tracking skill: `.learn 2580` (mining), `.learn 2383` (herbs), `.learn 2481` (treasure)
- `.tele bloodmyst`
- `.mod speed 10` (or faster)
- `.gm fly on`
- Fly around looking for minerals/herbs/treasure

## Known Issues and TODO List:

- Even though I checked all locations, I cannot guarantee some NPC isn't spawning inside a node. Hopefully we are at worst talking minor adjustments.

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1614780387938120077.sql [hunk 1]
```diff
@@ -0,0 +1,83 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1614780387938120077');
+
+-- Rewrite positions for treasures for zone: Bloodmyst Isle
+
+-- Before: 0 locations
+-- After: 28 locations
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1614780387938120077.sql:6]: https://github.com/azerothcore/azerothcore-wotlk/pull/4689#discussion_r586632432

Comment (Zoidwaffle) [reply data/sql/updates/pending_db_world/rev_1614780387938120077.sql]: Same :)

**Review by Si1ker — APPROVED** [review]

**Comment by Zoidwaffle** [comment]
I visited all the treasure spots - there should be 10 treasure, I believe I managed to count 10. If you spawn inside a chest or in cramped areas, it can be a bit hard to see.

https://www.youtube.com/watch?v=im2-mRj-Qs0

**Comment by Zoidwaffle** [comment]
@Si1ker : I'd rather not be my own tester, but I made a video showing that it works - would that count? Also, watch it at increased speed to check :grin:

So perhaps it's ready to merge unless others wants to validate it as well?

**Comment by FrancescoBorzi** [comment]
@Zoidwaffle the video is ok as testing proof

**Comment by Zoidwaffle** [comment]
bloodmyst_treasure

8 of 10 treasures found.

**Pull request merged**